thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -546,28 +509,6 @@ private Target onboardWithVendorThingID(
         return this.onboard(MediaTypes.MEDIA_TYPE_ONBOARDING_WITH_VENDOR_THING_ID_BY_OWNER_REQUEST, requestBody, vendorThingID, layoutPosition);
     }
 
-    /**
-     * On board IoT Cloud with the specified thing ID.
-     * When you are sure that the on boarding process has been done,
-     * this method is more convenient than
-     * {@link #onboard(String, String, String, JSONObject)}.
-     * If you are using a gateway, you need to use {@link #onboardEndnodeWithGateway(PendingEndNode, String)} instead.
-     * @param thingID Thing ID given by IoT Cloud. Must be specified.
-     * @param thingPassword Thing password given by vendor. Must be specified.
-     * @return Target instance can be used to operate target, manage resources
-     * of the target.
-     * @throws IllegalStateException Thrown when this instance is already onboarded.
-     * @throws ThingIFException Thrown when failed to connect IoT Cloud Server.
-     * @throws ThingIFRestException Thrown when server returns error response.
-     */
-    @NonNull
-    @WorkerThread
-    public Target onboard(
-            @NonNull String thingID,
-            @NonNull String thingPassword) throws
-            ThingIFException {
-        return onboardWithThingID(thingID, thingPassword, null);
```

Comment:
@yongpingchen 

I think that only mandatory parameter version of `onboard` methods may be useful for developers because setting `null` as an argument is little bit bother.

So leaving the methods or not depends on your and @satoshikumano san:+1:

If remove, LGTM. Please merge this.